e2e/features/stepDefinitions/service/overview.steps.ts

```diff
@@ -39,6 +39,46 @@ Then(/^I should (see|not see) the case details$/, async function (seeOrNotSee) {
     }
 });
 
+Then(/^I should (see|not see) the case details with (DC|RP) type$/, async function (seeOrNotSee, DeprivationOrRevocation) {
```

Comment:
Could we not expand DC/RP? As someone new or some non dev reading through the executed steps I can not determine the involved action here. After looking at I could still raise a question is RP - Revocation of Protection or Refusal of Protection. What's your opinion @Lee365?

Reply:
Agreed - expanded the DC/RP to be clear.

Reply:
Thanks @Lee365